neuralmonkey/decoders/sequence_classifier.py

```diff
@@ -23,18 +23,33 @@ def __init__(self,
                  encoders: List[Any],
                  vocabulary: Vocabulary,
                  data_id: str,
-                 layers: Optional[List[int]]=None,
-                 activation: Callable[[tf.Tensor], tf.Tensor]=tf.tanh,
+                 layers: List[int]=None,
+                 activation_fn: Callable[[tf.Tensor], tf.Tensor]=tf.nn.relu,
```

Comment:
nikdo to nepoužívá, takže tady OK, ale jinak prosím neměnit default parametry. Špatně se toho všímá a už jsem takhle jednou vytuhnul na dva měsíce s experimentama, který najednou přestaly běhat.